webpack.config.js

```diff
@@ -2,6 +2,7 @@ const HtmlWebPackPlugin = require("html-webpack-plugin");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 module.exports = {
   entry: './src/javascripts/main.js',
+  devtool: "eval-source-map",
```

Comment:
you can actually remove this line.  Check the file but it looks like this line is one of the last lines in the file